Packs/ANYRUN/Integrations/ANYRUN/ANYRUN.py

```diff
@@ -215,8 +398,9 @@ def operate_on_dict(the_dict):
         raise TypeError(err_msg)
 
 
-def generate_dbotscore(response):
-    """Creates DBotScore object based on the contents of 'response' argument
+def generate_dbotscore(response: Dict) -> List:
```

Comment:
Please change the doc also.
```suggestion
def generate_dbotscore(response: Dict) -> List[CommandResults]:
```